Library/Formula/ccat.rb

```diff
@@ -0,0 +1,14 @@
+require "formula"
+
+class Ccat < Formula
+  homepage "https://github.com/jingweno/ccat"
+  url "https://github.com/jingweno/ccat/archive/v0.1.0.tar.gz"
+  sha1 "fc0a1a49ca81fc85b9d50f48d63182d507b4627c"
```

Comment:
Didn't know we're using `sha256` forward. How about existing formulas? Any plan to fix all of them?